operations/host_spawn.go

```diff
@@ -1100,7 +1101,22 @@ func buildRsyncCommand(opts rsyncOpts) (*jasper.Command, error) {
 		args = append(args, "--exclude", pattern)
 	}
 	if opts.makeRemoteParentDirs {
-		args = append(args, fmt.Sprintf(`--rsync-path=mkdir -p "%s" && rsync`, filepath.Dir(opts.remote)))
+		var parentDir string
+		if runtime.GOOS == "windows" {
```

Comment:
This checks the OS on the machine running the command, not the machine it will be run on right?

Reply:
Yes. One of the fun things about this implementation is that we have no idea if the remote machine is a Windows machine or not, but it will still work, because the binaries we use on Windows are the cygwin binaries, which all require Unix-style paths. For example, if you give a remote path of `/home/Administrator/my/rsync/directory` on the Windows host, it will correctly make the paths within the cygwin filesystem (since `mkdir` is cygwin `mkdir`).